scripts/README.md

```diff
@@ -0,0 +1,46 @@
+## Overview
+This directory contains scripts for various actions around developing and operating ZAC.
+
+## Included scripts 
+
+```
+┌─ docker
+│  │     
+│  └─ build-docker-image.sh       Build docker image
```

Comment:
I think better to use links here. Otherwise refactoring scripts will very quickly break this readme indeed.

Reply:
I'm not sure we should add links to individual scripts here because this will be quite hard to maintain. I think it's better to keep this readme very minimal and add individual small readme's to every subfolder? Then again, most scripts already have a good description inside of them and a good name so how are readme's going to help at all.. Most people don't read readme's very quickly anyway I feel..

Reply:
What I'm trying to do with this README is to list what these directories are for and what we have at our disposal. Something like inventory or table-of-contents.

I'll try to move the description level to "directory only". This should eliminate refactoring issues.

Not sure we need links here. If you're browsing in github you might as well just click the real link in the tree. And browsing in terminal is also just getting inside the directory.

Reply:
@edgarvonk check the "directory only" readme

Reply:
>Not sure we need links here

The advantage of links is that they will be automatically updated if you change a script location or name. At least in IntelliJ. Big advantage if you ask me. :-)